src/components/navigation/Navigation.js

```diff
@@ -20,36 +22,49 @@ const mapStateToProps = (state) => {
   };
 };
 
+const mapDispatchToProps = (dispatch) => ({
+  onDragEnd: ({ type, source, destination }) => {
+    if (!destination) {
+      return;
+    } else if (type === 'page') {
```

Comment:
I think it would be better to wrap the two type checks into a `if (destination { ... }`